site/src/pages/OrganizationSettingsPage/IdpSyncPage/IdpSyncPage.tsx

```diff
@@ -105,6 +138,10 @@ export const IdpSyncPage: FC = () => {
 							groupsMap={groupsMap}
 							roles={rolesQuery.data}
 							organization={organization}
+							onGroupSyncFieldChange={handleGroupSyncFieldChange}
+							onRoleSyncFieldChange={handleRoleSyncFieldChange}
```

Comment:
you don't even need the extra functions here, just...
```suggestion
							onGroupSyncFieldChange={setGroupClaimField}
							onRoleSyncFieldChange={setRoleClaimField}
```

and then remove the definitions from above